lib/bolt/config/transport/local.rb

```diff
@@ -14,7 +14,7 @@ class Local < Base
           tmpdir
         ].freeze
 
-        OPTIONS = WINDOWS_OPTIONS.dup.concat(RUN_AS_OPTIONS).sort.freeze
+        OPTIONS = WINDOWS_OPTIONS.dup.concat(RUN_AS_OPTIONS).concat(%w[stream]).sort.freeze
```

Comment:
Is there a reason to not have this available to the `local` transport on Windows controllers?

Reply:
Oh no I just forgot!